runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -246,18 +253,22 @@ private void onMenuOptionClicked(MenuOptionClicked ev)
 				case MENUOP_QUICKGUIDE:
 					quickguide = "/Quick_guide";
 					//fallthrough;
+				case MENUOP_WIKI:
+					if (isSkill)
+					{
+						Matcher skillRegex = WikiPlugin.SKILL_REGEX.matcher(Text.removeTags(ev.getMenuTarget()));
+
+						if (skillRegex.find())
+						{
+							LinkBrowser.browse(WIKI_BASE + "/w/" + URLEncoder.encode(skillRegex.group(1)) + "?" + UTM_PARAMS);
+						}
+					}
+					//Diaries will fallthrough;
```

Comment:
Looks like a skill lookup will fall through here too and open 2 wiki pages? Is isSkill required? If skillRegex doesn't match this shouldn't do anything.

Reply:
Yes I forgot to add the break statement inside the if. Fixing it now